specification/network/resource-manager/Microsoft.Network/stable/2018-07-01/loadBalancer.json

```diff
@@ -1509,12 +1526,12 @@
           },
           "description": "Defines an external port range for inbound NAT to a single backend port on NICs associated with a load balancer. Inbound NAT rules are created automatically for each NIC associated with the Load Balancer using an external port from this range. Defining an Inbound NAT pool on your Load Balancer is mutually exclusive with defining inbound Nat rules. Inbound NAT pools are referenced from virtual machine scale sets. NICs that are associated with individual virtual machines cannot reference an inbound NAT pool. They have to reference individual inbound NAT rules."
         },
-        "outboundNatRules": {
+        "outboundRules": {
```

Comment:
Any reason why we want to change the type name here? This will force client applications to update their code from `new OutboundNatRule()` to `new OutboundRule`  when migrate from old api-version to this new one. Is this necessary?  Or the old name is just wrong?

Reply:
Yes, this is name change for new feature which is approved by dev and pm team

Reply:
So , functionality wise, I assume the outbound rules here will be a super set of the old NAT rules?
CC @tjprescott

Reply:
@yugangw-msft NAT rules in load balancer context implies port forwarding (inbound NAT rule). These rules specify outbound translations and have nothing to do with port forwarding. To avoid this confusion, we opted for removing Nat from the rule name and not reuse what was begun 2+ years ago and never finished and never released. 

Yes, OutboundRules are similar in shape, but different in how they are modeled.

OutboundNatRules were never available to anyone.

Reply:
Thanks for the clarification. This is a zero impact breaking change.